### PR TITLE
Add a `setup.py` file.

### PR DESCRIPTION
**Pull request by U039b**
That will allow users to install ApkLeaks by using the following command: 
```
pip install https://github.com/dwisiswant0/apkleaks
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from setuptools import setup
+
+install_requires = open('requirements.txt', 'r').readlines()
+
+setup(
+    name='apkleaks',
+    version='2.0.8',
```
Comment (dwisiswant0) [inline setup.py:7]: Is it possible to read from `VERSION` file instead?

Comment (U039b) [reply setup.py]: Please check the latest commit :)

**Review by dwisiswant0 — APPROVED** [review]
LGTM! Again, thanks for your contributions. You rock! \o/

**Comment by dwisiswant0** [comment]
Hey, @U039b. Install passed, but when I ran, error occurs: `ModuleNotFoundError: No module named 'apkleaks.apkleaks'; 'apkleaks' is not a package`.

**Comment by dwisiswant0** [comment]
I'm holding this on `pypi` branch.

**Comment by U039b** [comment]
Hi! 
Sorry, I did a mistake, I am fixing it.

**Pull request merged**